packages/cryptoplease/lib/features/profile/menu_screen.dart

```diff
@@ -9,19 +9,30 @@ class MenuScreen extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) => RefreshBalancesWrapper(
-        builder: (context, _) => CpTheme.light(
-          child: ColoredBox(
-            color: Colors.white,
-            child: SafeArea(
-              bottom: false,
-              child: ColoredBox(
-                color: CpColors.lightGreyBackground,
-                child: ListView(
-                  physics: const ClampingScrollPhysics(),
-                  children: const [
-                    MenuHeader(),
-                    MenuSection(),
-                  ],
+        builder: (context, onRefresh) => CpTheme.light(
+          child: RefreshIndicator(
```

Comment:
When scrolling down, there's white area beneath grey background, which looks weird:

https://user-images.githubusercontent.com/853356/191589342-c5947e8b-cebb-4efd-a52e-8b3bfafcaf3d.mp4

Reply:
Updated background color to match portions where it overscrolls. Is this okay?


https://user-images.githubusercontent.com/28533130/191645866-66edda96-0789-4499-9a8a-c427a50d4cfb.mp4